### PR TITLE
Move to XTT v0.9.2, with Ed25519->ECDSA change

### PR DESCRIPTION
**Pull request by zanebeckwith**

### CODE REVIEW

File: _build_ffi.py [hunk 1]
```diff
@@ -230,28 +233,28 @@
...
+                                                 const xtt_daa_group_pub_key_lrsw *gpk);
 
     xtt_return_code_type
     xtt_initialize_client_group_context_lrsw(struct xtt_client_group_context *ctx_out,
-                                             xtt_group_id *gid,
-                                             xtt_daa_priv_key_lrsw *priv_key,
-                                             xtt_daa_credential_lrsw *cred,
-                                             const unsigned char *basename,
-                                             uint16_t basename_length);
+                                    xtt_group_id *gid,
```
Comment (drbild) [inline _build_ffi.py]: bad indentation

File: _build_ffi.py [hunk 2]
```diff
@@ -262,28 +265,28 @@
...
     xtt_return_code_type
-    xtt_get_clients_longterm_key_ed25519(xtt_ed25519_pub_key *longterm_key_out,
-                                         const struct xtt_server_handshake_context *handshake_context);
+    xtt_get_clients_longterm_key_ecdsap256(xtt_ecdsap256_pub_key *longterm_key_out,
+                                           const struct xtt_server_handshake_context *handshake_context);
 
     xtt_return_code_type
     xtt_get_clients_identity(xtt_identity_type *client_id_out,
-                             const struct xtt_server_handshake_context *handshake_context);
+                              const struct xtt_server_handshake_context *handshake_context);
```
Comment (drbild) [inline _build_ffi.py]: bad indentation

File: _build_ffi.py [hunk 3]
```diff
@@ -262,28 +265,28 @@
...
     xtt_return_code_type
-    xtt_get_my_longterm_private_key_ed25519(xtt_ed25519_priv_key *longterm_key_priv_out,
-                                            const struct xtt_client_handshake_context *handshake_context);
+    xtt_get_my_longterm_private_key_ecdsap256(xtt_ecdsap256_priv_key *longterm_key_priv_out,
+                                              const struct xtt_client_handshake_context *handshake_context);
 
     xtt_return_code_type
     xtt_get_my_identity(xtt_identity_type *client_id_out,
-                        const struct xtt_client_handshake_context *handshake_context);
+                         const struct xtt_client_handshake_context *handshake_context);
```
Comment (drbild) [inline _build_ffi.py]: bad identation

File: xtt/test/context_test.py [hunk 4]
```diff
@@ -57,7 +57,7 @@ class TestServerCookieContext(unittest.TestCase):
     def test_init_succeeds(self):
         ctx = xtt.ServerCookieContext()
 
-class TestServerED25519CertificateContext(unittest.TestCase):
+class TestServerECDSAP256CertificateContext(unittest.TestCase):
```
Comment (drbild) [inline xtt/test/context_test.py:60]: I write super useful tests...

**Review by drbild — COMMENTED** [review]
Some indentation issues, but otherwise looks good to me if the tests pass.

**Comment by zanebeckwith** [comment]
Yep, tests pass.

Does it automatically push to PyPi once it's merged, or should I do that manually?

**Comment by drbild** [comment]
It should upload automatically.

If not, let me know.  I'd have to push it, because I don't think you have permission.

**Comment by zanebeckwith** [comment]
PyPi is still showing just `0.7.2.post1` as the latest

**Comment by drbild** [comment]
Ah, it only uploads to travis on tags.

I messed up the review.  The version bump should have gone in a commit separate from this PR. (Not every PR should result in a new release... Releasing is distinct action from adding features or fixing bugs...)

I just added a tag (on an empty release commit), so now PyPi should get the new version.

**Comment by zanebeckwith** [comment]
Gotcha. Make sense. Thanks!

**Pull request merged**